BUILD.md

```diff
@@ -0,0 +1,42 @@
+# Building barnabas
+
+Barnabas is currently using `make` as its main build system. Our make build supports several different targets mainly for building and pushing docker images.
+
+<!-- TOC depthFrom:2 -->
+
+- [Docker images](#docker-images)
+    - [Building Docker images](#building-docker-images)
+    - [Tagging and pushing Docker images](#tagging-and-pushing-docker-images)
+- [Building everything](#building-everything)
+- [Release](#release)
+
+<!-- /TOC -->
+
+## Docker images
+
+### Building Docker images
+
+`docker_build` target will build the Docker images provided by the barnabas project. You can build all Barnabas Docker images by calling `make docker_build` from the root of the Barnabas repository. Or you can build an individual Docker image by running `make docker_build` from the subdirectories with their respective dockerfile files - e.g. `kafka_base`, `kafka_statefulsets` etc.
+
+`docker_build` target will always build the images under the `enmasseproject` organization. This is necessary in order to be able to reuse the base image you might have just built without modifying all Dockerfile files.
+
+To configure `docker_build` target you can set environment variables:
+* `DOCKER_ORG` should contain the docker organization for tagging / pushing the images (default is your username)
```

Comment:
"(default is your username)" is ambiguous: My dockerhub username, or my username on the local machine? "(defaults to the value of the `$USER` environment variable, would be less ambiguous).

Did we standardise on surrounding `/` with spaces? That's not normal usage that I'm aware of.